gradle.properties

```diff
@@ -1,10 +1,10 @@
 projectVersion=3.2.1-SNAPSHOT
 projectGroup=io.micronaut.jaxrs
 micronautDocsVersion=2.0.0
-micronautVersion=3.3.4
+micronautVersion=3.4.3
```

Comment:
if you are updating the micronaut version this needs to go in the next minor of the module. You will need to setup projectVersion to 3.3.0-SNAPSHOT and githubCoreBranch to the next minor of core 3.5.x at this moment.

Reply:
Thanks for the info. I think this should be in a minor not a patch release, it's both new functionality and modifies existing stable code.